recipes/ulid/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(test_package CXX)
+
+find_package(ulid REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE ulid::ulid)
```

I think that would be the correct target name once the set_property call for CMakeDeps is added I commented about.
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE ULID::ulid)
```